src/main/java/school/lemon/changerequest/java/introduction/pr1/ConsoleCalculator.java

```diff
@@ -1,9 +1,77 @@
 package school.lemon.changerequest.java.introduction.pr1;
 
 
+import java.util.Scanner;
+
 public class ConsoleCalculator {
 
     public static void main(String[] args) {
 
+        boolean go = true;
+        Scanner userInput = new Scanner(System.in);
+        operationsChoise();
+        makeChoise();
+
+        while (go) {
+
+            String command1 = userInput.nextLine(); // поле
+
+            if (command1.equals("exit")) {
+                System.out.println("Bye-bye");
+                userInput.close();
+                break;
+
+            } else if (command1.equals("help")) {
+                operationsChoise();
+                makeChoise();
+                continue;
+            }
+
+            System.out.println("Enter first number:");
+            int x = userInput.nextInt();
+
+            System.out.println("Enter second number:");
+            int y = userInput.nextInt();
+            userInput.nextLine();
+
+            switch (command1) {
```

Comment:
I'd prefer to move this `switch` into some separate method. Just to keep loop pretty straightforward.